test/integration/replicaset/replicaset_test.go

```diff
@@ -930,3 +931,166 @@ func TestFullyLabeledReplicas(t *testing.T) {
 		t.Fatalf("Failed to verify only one pod is fully labeled: %v", err)
 	}
 }
+
+func getOrphanOptions() *metav1.DeleteOptions {
```

Comment:
We should be passing empty `DeleteOptions` because we want to test the default behavior when it's left unspecified.